ARMDeploy/ohbooking.json

```diff
@@ -46,6 +46,7 @@
         "serviceBusTopicNotificationsName": "mailnotifications",
         "serviceBusTopicUpdatesName": "mailupdates",
         "serviceBusQueueNotificationResponsesName": "mailidresponses",
+        "serviceBusQueueBookingSlot": "newbookingslot",
```

Comment:
It's not entirely clear what this name means, can you rename it so that we have some idea what it is used for ?